apps/moose-console/src/app/types.ts

```diff
@@ -57,12 +57,18 @@ export interface DataModel {
 }
 
 export interface MooseEnum {
-  Enum: {
-    name: string;
-    values: string[];
-  };
+  name: string;
+  values: MooseEnumMember[];
 }
 
+export interface MooseEnumMember {
+  name: string;
+  value?: MooseInt | MooseString;
+}
+
+export type MooseInt = { Int: number };
+export type MooseString = { String: string };
```

Comment:
why the "tag" instead of simply using union type?

Reply:
More readable function signatures